docs/visualization/visualize_unit_responses.ipynb

```diff
@@ -83,7 +83,7 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    colors = plt.cm.viridis(np.linspace(0, 1, len(np.transpose(units["waveform_mean"][unit_idx])) ))
</pre><p>Same than previous plot. An image might be more informative. Having the time as a colorer is a bit confusing to me. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>